massa-bootstrap/src/tests/mock_establisher.rs

```diff
@@ -2,6 +2,7 @@
 
 use massa_models::config::{CHANNEL_SIZE, MAX_DUPLEX_BUFFER_SIZE};
 use massa_time::MassaTime;
+use socket2 as _;
```

Comment:
Isn't this line useless?

Reply:
When using the testing features, the socket2 crate is not used and the unused_dependancy breaks the CI. As the rust features are additive, I don't see a way around this? Apart from just allowing the lint.

Reply:
ok, I forgot the CI